redis/commands/core.py

```diff
@@ -1069,6 +1069,11 @@ def wait(self, num_replicas, timeout, **kwargs):
         """
         return self.execute_command("WAIT", num_replicas, timeout, **kwargs)
 
+    def failover(self):
```

Comment:
Can we add a docstring indicating the behaviour. This will help users recognize we have completeness - but not use the function.

Reply:
Also - can we add a test to validate the exception is thrown?